packages/hdwallet-ledger/src/bitcoin.ts

```diff
@@ -115,6 +119,8 @@ export async function btcGetAddress (transport: LedgerTransport, msg: BTCGetAddr
 
  */
 export async function btcSignTx (wallet: BTCWallet, transport: LedgerTransport, msg: BTCSignTx): Promise<BTCSignedTx> {
+  await transport.open()
```

Comment:
would it be better to sink the `open()`/`close()` into `transport.call()` so you don't have to remember to add them everywhere?

Reply:
I tried to do this originally, but there seems to be issues with the user click event bubbling up to that point. I am going to get it working at the HDWallet level for now since it doesn't throw a dom exception at that point and then see what can be done to hoist that further up.

Reply:
ok, SGTM